package.json

```diff
@@ -11,23 +11,14 @@
   "scripts": {
     "start": "parallelshell 'bundle exec jekyll serve --watch --baseurl ''' 'gulp watch'",
     "build": "gulp build",
-    "cf-link": "npm link cf-core && npm link cf-buttons && npm link cf-pagination && npm link cf-typography && npm link cf-layout"
+    "cf-link": "npm link cf-core && npm link cf-buttons && npm link cf-pagination && npm link cf-typography && npm link cf-layout && npm link cf-icons"
   },
   "license": "CC0",
   "keywords": [
     "capital-framework"
   ],
   "dependencies": {
-    "cf-buttons": "latest",
-    "cf-core": "latest",
-    "cf-expandables": "latest",
-    "cf-forms": "latest",
-    "cf-grid": "latest",
-    "cf-icons": "latest",
-    "cf-layout": "latest",
-    "cf-pagination": "latest",
-    "cf-tables": "latest",
-    "cf-typography": "latest",
+    "capital-framework": "latest",
```

Comment:
Uh, I actually don't know why this changed, I don't remember doing it myself, maybe something automated in a gulp script???